contracts/mocks/TimedCrowdsaleImpl.sol

```diff
@@ -1,14 +1,19 @@
 pragma solidity ^0.5.2;
 
+import "../ownership/Ownable.sol";
 import "../token/ERC20/IERC20.sol";
 import "../crowdsale/validation/TimedCrowdsale.sol";
 
-contract TimedCrowdsaleImpl is TimedCrowdsale {
+contract TimedCrowdsaleImpl is TimedCrowdsale, Ownable {
```

Comment:
There's no need to make the mock `Ownable`, all we want to test is how the internal function works.